### PR TITLE
Calvin's replication

### PR DESCRIPTION
**Pull request by printfCalvin**
Ran both experiments on Ubuntu in Windows. No problem occurred.

### CODE REVIEW

**Comment by lintool** [comment]
Invalid commit: This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Also, please revert `collections/.gitkeep`

**Comment by printfCalvin** [comment]
I have reverted `collections/.gitkeep`. I copied the files in collections directly from anserini. But I don't quite understand this sentence `Invalid commit: This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.` I do don't think I have the permission to

File: docs/experiments-msmarco-doc.md [hunk 1]
```diff
@@ -136,3 +136,4 @@ We can see that Anserini's (tuned) BM25 baseline is already much better than the
 + Results reproduced by [@yemiliey](https://github.com/yemiliey) on 2021-01-18 (commit [`98f3236`](https://github.com/castorini/pyserini/commit/98f323659c8a0a5d8ef26bb3f6768458a34e3eb9))
 + Results reproduced by [@larryli1999](https://github.com/larryli1999) on 2021-01-04 (commit [`74a87e4`](https://github.com/castorini/pyserini/commit/74a87e4951c98d7b066273140576d3cccd9ea0ed))
 + Results reproduced by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-01-04 (commit [`7261223`](https://github.com/castorini/pyserini/commit/72612232bc886e71e8de9431a899a7c68f1d82c7))
++ Results reproduced by [@printfCalvin](https://github.com/printfCalvin) on 2021-04-11 (commit [`4fcdeeb`](https://github.com/castorini/pyserini/commit/4fcdeebc26956fb93adfd0a961dd3c5e42de4448))
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:139]: If you go to this URL above, you'll see the error: This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Please point this link to the commit id where you ran the reproduction experiments.

Comment (printfCalvin) [reply docs/experiments-msmarco-doc.md:139]: Ok. I see. Sorry about that. I will change it right now.

File: collections/.gitkeep [hunk 2]
```diff
@@ -1 +1 @@
-# This is the default directory for document collections. Placeholder so that directory is kept in git.
\ No newline at end of file
+# This is the default directory for document collections. Placeholder so that directory is kept in git.
```
Comment (lintool) [inline collections/.gitkeep:1]: revert please.

File: docs/experiments-msmarco-doc.md [hunk 3]
```diff
@@ -136,3 +136,4 @@ We can see that Anserini's (tuned) BM25 baseline is already much better than the
 + Results reproduced by [@yemiliey](https://github.com/yemiliey) on 2021-01-18 (commit [`98f3236`](https://github.com/castorini/pyserini/commit/98f323659c8a0a5d8ef26bb3f6768458a34e3eb9))
 + Results reproduced by [@larryli1999](https://github.com/larryli1999) on 2021-01-04 (commit [`74a87e4`](https://github.com/castorini/pyserini/commit/74a87e4951c98d7b066273140576d3cccd9ea0ed))
 + Results reproduced by [@ArthurChen189](https://github.com/ArthurChen189) on 2021-01-04 (commit [`7261223`](https://github.com/castorini/pyserini/commit/72612232bc886e71e8de9431a899a7c68f1d82c7))
++ Results reproduced by [@printfCalvin](https://github.com/printfCalvin) on 2021-04-11 (commit [`16131e5`](https://github.com/castorini/pyserini/commit/16131e5e584831c876c329d53345ab6fe219d4a8))
```
Comment (lintool) [inline docs/experiments-msmarco-doc.md:139]: still same issue...

**Review by lintool — CHANGES_REQUESTED** [review]

File: docs/experiments-msmarco-doc.md (hunk 3 above)
Comment (printfCalvin) [reply docs/experiments-msmarco-doc.md:139]: I konw. Trying to figure out what went wrong right now.

**Comment by printfCalvin** [comment]
sorry about the number of commits. I think I misunderstood the replication log.

**Review by lintool — CHANGES_REQUESTED** [review]
commit links are good now, please revert `collections/.gitkeep`

**Comment by printfCalvin** [comment]
My local version does not have the new line char. But somehow the github repo has it. Can I close this pull request, and start a new one?

**Comment by printfCalvin** [comment]
Ok, thanks a lot.

**Pull request closed without merging**